cmake.html.markdown

```diff
@@ -0,0 +1,176 @@
+---
+language: cmake
+contributors:
+    - ["Bruno Alano", "https://github.com/brunoalano"]
+filename: CMake
+---
+
+CMake it's a cross-platform, open-source build system. This tool will allow you
```

Comment:
Oh, also if you could move this to the "tools" section by adding the following:

```
category: tool
tool: tmux
```

This should replace the `language` specifier in the heading to ensure it ends up in the right spot.